### PR TITLE
DOC: Fix some incorrect markups

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: numpy/core/code_generators/ufunc_docstrings.py [hunk 1]
```diff
@@ -1047,8 +1047,8 @@ def add_newdoc(place, name, doc):
 
     References
     ----------
-    .. Wikipedia, "Heaviside step function",
-       https://en.wikipedia.org/wiki/Heaviside_step_function
+    Wikipedia, "Heaviside step function",
+    https://en.wikipedia.org/wiki/Heaviside_step_function
```
Comment (rossbar) [inline numpy/core/code_generators/ufunc_docstrings.py:1051]: Just a nit, but according to the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#references), the references are supposed to be enumerated *and* refer back to the main text. Therefore I vote for one of two courses of action: 1) Remove the references section here entirely or 2) change this to

```suggestion
    .. [1] Wikipedia, "Heaviside step function",
       https://en.wikipedia.org/wiki/Heaviside_step_function
```

And add `[1]_` to L1021.

**Review by rossbar — COMMENTED** [review]
All the rst formats look good, just one nit about conforming to the docstring standard re: references.

File: numpy/core/_add_newdocs.py [hunk 2]
```diff
@@ -6476,11 +6476,14 @@
...
     Attributes
     ----------
-    Note: once a busdaycalendar object is created, you cannot modify the
-    weekmask or holidays.  The attributes return copies of internal data.
     weekmask : (copy) seven-element array of bool
     holidays : (copy) sorted array of datetime64[D]
 
+    Notes
+    -----
+    once a busdaycalendar object is created, you cannot modify the
```
Comment (mattip) [inline numpy/core/_add_newdocs.py:6484]:
```suggestion
    Once a busdaycalendar object is created, you cannot modify the
```

File: numpy/core/code_generators/ufunc_docstrings.py [hunk 3]
```diff
@@ -1018,7 +1018,7 @@ def add_newdoc(place, name, doc):
     """
     Compute the Heaviside step function.
 
-    The Heaviside step function is defined as::
+    The Heaviside step function [1]_ is defined as::
```
Comment (mattip) [inline numpy/core/code_generators/ufunc_docstrings.py:1021]: The footnote in the [rendered doc](https://output.circle-artifacts.com/output/job/198862a1-ea91-445f-bce3-e200e584b975/artifacts/0/doc/build/html/reference/generated/numpy.heaviside.html#id2) seems broken.

**Review by mattip — COMMENTED** [review]
Thanks! Two more tiny nits

**Review by rossbar — APPROVED** [review]
LGTM, thanks @F3eQnxN3RriK & @mattip !

**Pull request merged**